ui/js/content.js

```diff
@@ -290,10 +290,10 @@ require(['jquery', 'underscore', 'oae.core'], function($, _, oae) {
      * @see manageaccess#initManageAccess
```

Comment:
I thought we decided to have editors only able to give `viewer` roles to others and leave `manager` as the default role.